api-reference/50 Common/Object Structures/PdfExportDataGridProps/topLeft/topLeft.md

```diff
@@ -4,7 +4,7 @@ type: Object
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a starting position for DataGrid export on the page. Contains **x** and **y** properties. You can locate this position only below the page margins.
```

Comment:
Specifies a start position for DataGrid export on the page. Contains **x** and **y** properties. You can locate this position only below the page margins.